1_fetch/out/model_RMSEs.csv

```diff
@@ -0,0 +1,116 @@
+exper_n,exper_id,model_type,rmse
```

Comment:
Okay, thanks. I may have misunderstood the directions for the out folders. Do you normally commit these folders, or generate them within the script (`dir.create`)? I did not see a way to commit an empty out directory in git (at least with command line), so that's what led me to commit the data within it. Maybe that's possible with the GitHub interface?

Reply:
Good Q. We (or at least I) have been inconsistent in our approach for `out` dirs - sometimes I think we just assume that others on the team who fork the repo will know to create `out` directories as needed, so here's your heads-up that you might see that sometimes. But it's better to commit the folders or include a `dir.create` within the script, as you suggested.

Either of those approaches can work. I assume you know how you'd use `dir.create` already. You can't commit a totally empty directory, but you can commit a nearly-empty directory whose only contents are an empty file, e.g., something named `placeholder.txt`.